project/phpunit.xml.dist

```diff
@@ -23,16 +23,16 @@ It's auto-generated by sonata-project/dev-kit package.
         </testsuite>
     </testsuites>
 
-    <listeners>
-       <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener" />
-    </listeners>
-
     <filter>
         <whitelist>
             <directory suffix=".php">./src/</directory>
         </whitelist>
     </filter>
 
+    <listeners>
+        <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener"/>
```

Comment:
Well. It is just a move?

Reply:
Yes I have an open Todo and wanted to have some feedback :)